db/db.go

```diff
@@ -294,10 +295,10 @@ func (d *DB) Close() error {
 	}
 	d.closed = true
 
-	if d.adapter != nil {
-		d.adapter.Close()
+	if d.connector != nil {
+		_ = d.connector.Close()
```

Comment:
Nit: This was the case before this PR, but we could log the possible error? Maybe the right thing is returning it after trying to close things later, I'm not sure. But an error closing the `connector` sounds like an interesting thing to know about.

Reply:
Yep, I'll just return the error.